docs/books/admin_guide/05-vi.md

```diff
@@ -204,37 +204,37 @@ VI switches to *insert* mode after entering one of these keys.
 
 !!! Note
 
-    VI switches to *insertion* mode. So you will have to press the <kbd>ESC</kbd> key to return to *command* mode.
+    VI switches to *insertion* mode. So you will have to press the ++ESC++ key to return to *command* mode.
 
 ### In relation to a character
 
 * Inserting text before a character:
 
-<kbd>i</kbd> (*insert*)
+++i++ (*insert*)
 
```

Comment:
I think I'm going to have to just merge this and then re-edit it at some point. The problem is the "intent" of the document (should the command show up as a lowercase? If so, it needs to be in quotes)... I won't be able to deal with this though for a while, so I may merge it and just edit after the fact. Unless you want to keep working on it @nishaaaaaant. There's no rush as far as I'm concerned.

Reply:
You can merge the changes that are there up to now. 
I will edit the document with all the new changes in place. Go please go ahead and merge this.